nextcord/ext/commands/bot.py

```diff
@@ -669,8 +674,18 @@ def _load_from_module_spec(self, spec: importlib.machinery.ModuleSpec, key: str)
             del sys.modules[key]
             raise errors.NoEntryPointError(key)
 
+        params = inspect.signature(setup).parameters
+        has_kwargs = len(params) > 1
+
+        if extras:
```

Comment:
Someone could provide a falsy object and `TypeError` wouldn't be raised, didn't notice this earlier
```suggestion
        if extras is not None:
```